tests/abort_dep_tasks.py

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 from __future__ import print_function
-import ecflow
+import ecflow as ecflow
```

Comment:
Why is this required?

Reply:
I was getting some errors on Acorn, maybe something was wrong with ecflow installation. They keep changing things, I'll test again and remove it if it's not necessary anymore.

Reply:
Looks like they installed the latest version of ecflow the same day, or day before I made this change:
```
$ ls -dl /apps/ops/prod/nco/core/ecflow.v5.6.0.*
drwxr-xr-x 8 ops.prod prod 87 Apr 14 00:08 /apps/ops/prod/nco/core/ecflow.v5.6.0.3
drwxr-xr-x 8 ops.prod prod 87 Aug  5 16:15 /apps/ops/prod/nco/core/ecflow.v5.6.0.4
drwxr-xr-x 8 ops.prod prod 87 Aug 17 15:39 /apps/ops/prod/nco/core/ecflow.v5.6.0.5
drwxr-xr-x 8 ops.prod prod 87 Sep  2 19:58 /apps/ops/prod/nco/core/ecflow.v5.6.0.6
```
Who knows what the differences are between 5.6.0.5 and 5.6.0.6.